### PR TITLE
Chore/update scripts & docs

### PR DESCRIPTION
**Pull request by Grkmk**
This PR will update various scripts & docs to reflect the project's current state.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,18 +13,30 @@ cd client && npm install && cd server && npm install && cd ..
...
 ```bash
 cd client && npm start # To start the frontend
-cd server && npm start:debug # To start the backend
+cd server && npm start:services # To start the nest application
+
+# wait until nest app is running
 cd client && npm run codegen # To generate graphql stuff
+
+cd server && npm run codegen # To generate graphql stuff
+cd server && npm start:debug # To start the backend
```
Comment (dirkkok) [inline README.md:28]: This must go before the codegen stuff, right?

File: client/package.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
         "zxcvbn": "^4.4.2"
     },
     "scripts": {
-        "start": "react-scripts start",
+        "start": "react-scripts start && npm run compile-translations",
```
Comment (dirkkok) [inline client/package.json:45]: Does it work in this order? I'd assume compile would have to run before `react-scripts start`

**Review by dirkkok — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Grkmk) [reply README.md]: ah yes, for some reason i kept thinking that the `:services` was starting the nest app. fixed

File: client/package.json (hunk 2 above)
Comment (Grkmk) [reply client/package.json]: yea it was but it makes more sense to start the compiler before running the script since it may trigger a restart. fixed

**Review by Jorann — APPROVED** [review]
The itermocil stuff is weird.

**Review by dirkkok — APPROVED** [review]

**Pull request merged**